backend/clubs/migrations/0047_auto_20201023_1848.py

```diff
@@ -1,15 +1,28 @@
-# Generated by Django 3.1.2 on 2020-10-18 21:41
+# Generated by Django 3.1.2 on 2020-10-23 22:48
```

Comment:
Since the `0047_profile_uuid_secret` migration is already in production, we may not want to delete this migration.

It is much easier to delete our local development db than to modify the production db. Try to avoid modifying any migrations committed to `master`, as those are already in production.